### PR TITLE
Update scapy.rb

### PR DESCRIPTION
**Pull request by packeteer**
Update scapy build to include latest version (2.3.2), and pull from the new repo (github)

### CODE REVIEW

File: scapy.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Scapy < Formula
   desc "Powerful interactive packet manipulation program"
   homepage "http://www.secdev.org/projects/scapy/"
   url "https://bitbucket.org/secdev/scapy/downloads/scapy-2.3.1.zip"
```
Comment (tdsmith) [inline scapy.rb]: Did you mean to update the url as well?

Comment (packeteer) [inline scapy.rb]: yes I did :(

**Comment by tdsmith** [comment]
Thanks! Pulled with minor changes.

**Comment by packeteer** [comment]
sorry for making it so crap :)

**Comment by tdsmith** [comment]
Not at all! Much appreciated :)

**Comment by packeteer** [comment]
bugga, the sha is wrong, I'll raise another PR

**Comment by tdsmith** [comment]
Ugh duh; dumb mistake on my part. I'll fix it.

**Pull request closed without merging**